### DISCUSSION TITLE
Bogus scripts pre written, web redirects, Null. I need help badly

### DISCUSSION THREAD

**Discussion by AceArd86** [community/community#45150] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I have had problems for going on a year now driving myself crazy. I’ve had no support or help. I can’t do this alone I’ve figured out so I come here. I have a bug. I think I have many. I can’t get anyone to believe me and I don’t know where to begin. I believe my Unix epoch is off. I have files that are stuck in 1970. I believe that some “thing” is using account takeover. I believe that someone is using a super user against me. I have a json and JavaScript string in my code that is like 15000 lines long to figure out and multiple errors about, User agent, User app and some others. I can’t even download anything cause it goes against me. Locked out of multiple accounts and computers. The list goes on. Please any advice or direction would be so appreciated. I can send examples if I have to

**Comment by jge162** [comment]
Regarding the JSON and JavaScript string, it is important to check for any errors and bugs in the code. It's also important to make sure that you are using the latest version of the software and libraries.

Regarding the locked out of multiple accounts and computers, it could be caused by a security vulnerability or a malicious actor attempting to gain unauthorized access to your accounts.

I would recommend that you try to identify the specific problem and then seek help from experts in the field.

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for posting in the GitHub Community, @AceArd86 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
